DD/Views/Partials/homePage/_navigation.cshtml

```diff
@@ -18,46 +17,40 @@
 <head>
     @await Html.PartialAsync("~/Views/Partials/_metaData.cshtml")
     <link rel="stylesheet" href="~/Styles/navigation.css" />
+
 </head>
 <body>
     <nav id="navigation">
         <div>
+            <a for="check" class="open-menu"><i class="fas fa-bars"></i></a>
             <ul>
                 @if (allVisiblePages != null && allVisiblePages.Any())
+                {
+                    var dropdownId = 0;
+                    foreach (var item in allVisiblePages)
                     {
-                        var dropdownId = 0;
-                @foreach (var item in allVisiblePages)
+                        var hasChildLinks = item.Name == "Emprendimiento" || item.Name == "Capacitacion" || item.Name == "Ferias y Eventos" || item.Name == "Sector Industrial" || item.Name == "Apoyos y Financiamientos";
+                        if (hasChildLinks)
                         {
-                            var hasChildLinks = item.Name == "Emprendimiento" || item.Name == "Capacitacion" || item.Name == "Ferias y Eventos" || item.Name == "Sector Industrial" || item.Name == "Apoyos y Financiamientos";
-                @if (hasChildLinks)
-                            {
-                <li>
-                    <a href="@item.Url()">@item.Name</a>
-                    <ul>
-                        @foreach (var childLink in item.Children())
+                            <li>
+                                <a href="@item.Url()">@item.Name</a>
+                                <ul>
+                                    @foreach (var childLink in item.Children())
                                         {
-                        <li>
-                            <a href="@childLink.Url()">@childLink.Name</a>
-                            @*
-                                <ul >
-                                    @foreach (var thirdLink in childLink.Children())
-                                    {
-                                        <li >
-                                            <a href="@thirdLink.Url()">@thirdLink.Name</a>
-                                        </li>
-                                    }
-                                </ul>
-                            *@
-                        </li>
+                                    <li>
+                                        <a href="@childLink.Url()">@childLink.Name</a>
+                                        @*add dropdowns in navigationBar*@
+                                    </li>
                                         }
-                    </ul>
-                </li>
-                            }
-                            dropdownId++;
+                                </ul>
+                            </li>
                         }
+                        dropdownId++;
```

Comment:
enhance this code, avoid using dropdownId++ check with chat gpt for best practices

Reply:
I already changed the code, erasing the line 28 and 48 to avoid the counting dropdownId